### PR TITLE
Fix #3559 : Merged profile_edit_fragment.xml into single xml file

### LINKED ISSUE #3559: Merge profile_edit_fragment.xml into single xml file
Currently there are 2 versions of `profile_edit_fragment.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by mahmood199**
Phone portrait 
Before    
Screenshot_20211020_010509_org oppia android  
And After
Screenshot_20211020_011834_org oppia android

Phone landscape(2 pictures in each)(because to view full layout i had to scroll and therefore total 4 images)
Before 
Screenshot_20211020_013439_org oppia android
Screenshot_20211020_010528_org oppia android
And After
Screenshot_20211020_010525_org oppia android
Screenshot_20211020_013444_org oppia android

Tablet Portrait
Before 
Screenshot (27)
And After
Screenshot (28)

Tablet landscape
Before 
Screenshot (29)
and After
Screenshot (30)

In the previous Phone landscape there was a big spacing above and below deletion button and view margin misallignment. It is fixed in this PR.
Thank you. :)

### CODE REVIEW

**Comment by mahmood199** [comment]
Please merge this @rt4914 🙂

**Comment by FareesHussain** [comment]
@mahmood199 Thanks for showing interest in oppia. Please check out our [wiki](https://github.com/oppia/oppia-android/wiki#onboarding-instructions) first. This will help you in setting up the project and once you sign the CLA and fill the survey form we can assign you issues to work on.

**Comment by mahmood199** [comment]
@FareesHussain I have filled the form already.
Actually I had created a PR then got the prompt to fill the form....do i  need to create a PR again?

**Comment by FareesHussain** [comment]
@mahmood199 did you fill the contributors survey

**Comment by mahmood199** [comment]
Yes, just filled the form
Please check once

**Review by Arjupta — CHANGES_REQUESTED** [review]
I have took an initial pass over you PR and here are some points to work improve.
- Always try to remove file changes under ` .idea` directory from the commit.
- Mention the Issue number for which you have created this PR.
- The PR title should be in this format `Fix #issue_number: Description`

**Review by mahmood199 — COMMENTED** [review]
.idea files deleted

**Comment by mahmood199** [comment]
Me waiting for my PR to be merged.
[https://www.google.com/url?sa=i&url=https%3A%2F%2Ftenor.com%2Fview%2Fmr-bean-waiting-still-waiting-gif-13052487&psig=AOvVaw3sN4NiIOHbQH-Hi747sb4U&ust=1636435303853000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCPjRoNqCiPQCFQAAAAAdAAAAABAD](url)

**Comment by BenHenning** [comment]
@ARJUPTA per https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section please make sure that you assign your PR to whoever needs to look at it next (otherwise it's likely that we won't see it).

File: .idea/misc.xml
Comment (rt4914) [inline .idea/misc.xml:9]: We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.

File: .idea/runConfigurations.xml
Comment (rt4914) [inline .idea/runConfigurations.xml:11]: We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.

File: app/src/main/res/layout-land/profile_edit_fragment.xml
Comment (rt4914) [inline app/src/main/res/layout-land/profile_edit_fragment.xml:38]: Instead of one single dimens introduce two separate dimens as below:
profile_edit_fragment_profile_width
profile_edit_fragment_profile_height

File: app/src/main/res/values/dimens.xml
Comment (rt4914) [inline app/src/main/res/values/dimens.xml:581]: Rename this to `profile_edit_fragment_profile_image_elevation` and use it in all xml files.

File: app/src/main/res/values/dimens.xml
Comment (rt4914) [inline app/src/main/res/values/dimens.xml:603]: If a value is same in both the files then do not define it in `dimens.xml` file, instead hard code it in xml file only.
Apply this to all dimens that you have introduced.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@mahmood199 Thanks. PTAL

**Comment by BenHenning** [comment]
My review is only needed due to the .idea file changes. Please remove them from the PR & assign back to me if my review is still needed/wanted.

**Comment by Arjupta** [comment]
> @ARJUPTA per https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section please make sure that you assign your PR to whoever needs to look at it next (otherwise it's likely that we won't see it).

@BenHenning actually I was supposed to take a second pass but @mahmood199 forgot to ask my review because of which ther was delay. I would have surely assigned you guys after giving LGTM from my side.

**Comment by BenHenning** [comment]
No problem--thanks for the context @ARJUPTA. :)

**Comment by BenHenning** [comment]
FYI I'm listed as a codeowner for this PR & I'll be unavailable to perform code reviews over the next 2 weeks--thanks for your patience.

**Comment by rt4914** [comment]
Closing this PR and un-assigning because of inactivity @mahmood199

**Pull request closed without merging**
